MaxText/train.py

```diff
@@ -292,7 +294,8 @@ def dpo_loss_fn(model, config, data, dropout_rng, params, reference_params, is_t
   rejected_segmentation = data["rejected_segmentation"][..., 1:]
   n_logits = logits.shape[-3] // 2  # [B, S, E] - [batch, sequence, embedding/vocab]
   chosen_logits, rejected_logits = logits[:n_logits, :, :], logits[n_logits:, :, :]  # [B, S, E], [B, S, E]
-  chosen_ref_logits, rejected_ref_logits = ref_logits[:n_logits, :, :], ref_logits[n_logits:, :, :]  # [B, S, E], [B, S, E]
+  chosen_ref_logits, rejected_ref_logits = ref_logits[:n_logits, :, :], ref_logits[n_logits:, :, :]
+  # ^ [B, S, E], [B, S, E]
```

Comment:
can we put this comment before the previous line

Reply:
Done